nextcloud-jail.sh

```diff
@@ -208,20 +221,23 @@ elif [ "${DATABASE}" = "pgsql" ]; then
 fi
 iocage exec "${JAIL_NAME}" mkdir -p /mnt/includes
 iocage exec "${JAIL_NAME}" mkdir -p /usr/local/www/nextcloud/config
+mkdir -p "${JAILS_MOUNT}"/jails/${JAIL_NAME}/root/usr/local/www/nextcloud/themes
 mkdir -p "${JAILS_MOUNT}"/jails/${JAIL_NAME}/root/var/db/portsnap
 mkdir -p "${JAILS_MOUNT}"/jails/${JAIL_NAME}/root/mnt/files
 mkdir -p "${JAILS_MOUNT}"/jails/${JAIL_NAME}/root/mnt/includes
 mkdir -p "${JAILS_MOUNT}"/jails/${JAIL_NAME}/root/usr/ports
+
 iocage fstab -a "${JAIL_NAME}" "${PORTS_PATH}"/ports /usr/ports nullfs rw 0 0
 iocage fstab -a "${JAIL_NAME}" "${PORTS_PATH}"/db /var/db/portsnap nullfs rw 0 0
 iocage fstab -a "${JAIL_NAME}" "${FILES_PATH}" /mnt/files nullfs rw 0 0
 iocage fstab -a "${JAIL_NAME}" "${CONFIG_PATH}" /usr/local/www/nextcloud/config nullfs rw 0 0
+iocage fstab -a "${JAIL_NAME}" "${CONFIG_PATH}" /usr/local/www/nextcloud/themes nullfs rw 0 0
```

Comment:
This should use THEMES_PATH, not CONFIG_PATH

Reply:
Fixed, mistake during rebase